src/manager/app/routers/services.py

```diff
@@ -0,0 +1,72 @@
+from datetime import datetime
+
+from fastapi import APIRouter, Depends
+from pydantic import BaseModel, computed_field, validator
+
+from app.constants import DEFAULT_PROJECT_PREFIX
+from app.docker_client import get_docker_client
+
+service_router = r = APIRouter()
+
+
+class ServiceInfo(BaseModel):
+    id: str
+    name: str
+    created_at: str
+    started_at: str | None
+    updated_at: str | None
+    status: str
+    image: str
+    restart_count: int
+    exposed_ports: dict
+
+    @computed_field  # type: ignore
+    @property
+    def port(self) -> str | None:
+        main_port = self.exposed_ports.get("8000/tcp", [])
```

Comment:
what constant is 8000?

Reply:
I just removed the port information, we don't need it now.